site/docs/tools/third_party_tools.md

```diff
@@ -0,0 +1,6 @@
+# Third Party Tools
+
+## Substrait-tools
+The [substrait-tools](https://pypi.org/project/substrait-tools/) python package provides 
+a command line interface for producing/consuming substrait plans by leveraging the APIs
+from different producers and consumers. Source code can be found [here](https://github.com/richtia/substrait-tools).
```

Comment:
```suggestion
from different producers and consumers.
```

Let's leave the "source code can be found here" for pypi (as an aside, you can configure pypi to show a github link by using project links https://stackoverflow.com/questions/61156290/how-to-set-project-links-in-pypi).

Reply:
Cool, just configured it.